lib/util.js

```diff
@@ -63,6 +64,30 @@ const fetchTextFromURL = (listURL) => {
   return p
 }
 
+const generateComponentMetadata = (
+  inputDir,
+  publisherProofKey,
+  publisherProofKeyAlt
```

Comment:
are we using the existing crx signing keys for this?

for production, is this going to be called in CI?

Reply:
I'm not sure. I am inclined to generate a new one for this purpose.

Reply:
`Components Verified Contents PEM` key in 1Password